web/src/components/gnd-feature-type-editor/gnd-form-editor.js

```diff
@@ -48,6 +53,23 @@ const styles = (theme) => ({
   },
 });
 
+const DragHandle = sortableHandle(() => <span>::</span>);
+
+const SortableItem = sortableElement(({element, onChange}) => (
+  <div>
+    <DragHandle />
+    <GndFormElementEditor
+      key={element.id}
+      element={element}
+      onChange={onChange}
+    />
+  </div>
+));
+
+const SortableContainer = sortableContainer(({children}) => {
```

Comment:
According to [the docs](https://github.com/clauderic/react-sortable-hoc#sortablecontainer-hoc), we can also set `lockAxis` to `y` to ensure dragging happens only vertically.